lib/pageflow/entry_type.rb

```diff
@@ -4,18 +4,19 @@ module Pageflow
   # @since 15.1
   class EntryType
     # @api private
-    attr_reader :name, :frontend_app, :editor_fragment_renderer
+    attr_reader :name, :frontend_app, :editor_fragment_renderer, :configuration
 
     # @param name [String] A unique name.
     #
     # @param frontend_app [#call] A rack app that renders the entry
     #   when not in the editor (i.e. preview and published entries).
     #
     # @param editor_fragment_renderer
-    def initialize(name:, frontend_app:, editor_fragment_renderer:)
+    def initialize(name:, frontend_app:, editor_fragment_renderer:, configuration:)
```

Comment:
Parameter not yet documented in inline doc. Let's also add a section to the ["Creating entry types" guide](https://github.com/codevise/pageflow/blob/master/doc/creating_entry_types.md) describing this feature.